shuup_checkoutfi/locale/fi/LC_MESSAGES/django.po

```diff
@@ -36,17 +36,30 @@ msgstr "Checkout.fi"
 msgid "Missing values"
 msgstr "Arvoja puuttuu"
 
+msgid "Checkout.fi payment was canceled by the user."
+msgstr "Checkout.fi maksu peruutettiin käyttäjän toimesta"
+
 msgid "Payment canceled. Try again."
 msgstr "Peruit maksamisen. Yritä uudestaan."
 
+msgid "Order payment status marked as deferred."
+msgstr "Tilauksen maksutila muutettu viivästyneeksi."
+
 #, python-brace-format
 msgid "Checkout.fi {payment_id} (ref {order_reference})"
 msgstr "Checkout.fi {payment_id} (viite {order_reference})"
 
+msgid "Checkout.fi payment validation failed."
+msgstr "Checkout.fi maksun validointi epäonnistui."
+
 msgid "Payment validation failed."
 msgstr "Maksun validointi epäonnistui."
 
 #, python-format
+msgid "Checkout.fi returned unknown return code %s."
+msgstr "Checkout.fi palautti tuntemattoman paluuarvon %s"
```

Comment:
You're missing the final period. And the message could use a colon for clarity. Also use the new style format string (`... return unknown return code: {status}`) rather than `%s`.